ecs-cluster/alerts/issues/13000_FAILED_TIER_issue.yaml

```
@@ -0,0 +1,9 @@
+- description: "If tier is down or can not be written to"
```

what is a "tier" here?  storage tier?  if so reword it as:
```suggestion
- description: "Storage tier is not available or degraded"
```

---

Done